README.md

```diff
@@ -181,11 +180,11 @@ This module will use SemVer, and will stay on v0.X for the foreseeable future
 
 | Name | Description |
 |------|-------------|
-| cluster\_kubeconfig | KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
+| cluster\_kubeconfig | [SENSITIVE] KUBECONFIG yaml file contents to connect to the cluster. DO NOT USE unless you have no other options. Users should use the KUBECONFIG that Rancher provides to them rather than this. |
 | rancher\_admin\_password | Password for Rancher 'admin' user |
 | rancher\_admin\_token | API Token for Rancher 'admin' user |
 | rancher\_endpoint | Endpoint of Rancher Server |
-| ssh\_private\_key | Cluster nodes' private SSH key |
+| ssh\_private\_key | [SENSITIVE] Cluster nodes' private SSH key |
```

Comment:
Codacy found an issue: [[no-shortcut-reference-link] Use the trailing `[]` on reference links](https://app.codacy.com/gh/saic-oss/terraform-aws-rke-rancher-master-cluster/pullRequest?prid=6339862)